tools/gfx-unit-test/draw-instanced-test.cpp

```diff
@@ -86,8 +116,8 @@ namespace gfx_test
         renderPassDesc.renderTargetAccess = &renderTargetAccess;
         ComPtr<IRenderPassLayout> renderPass = device->createRenderPassLayout(renderPassDesc);
 
-        const int width = 2;
-        const int height = 2;
+        const int width = 256;
```

Comment:
Any particular reason to use 256 by 256? We didn't check the value of those pixels.

Reply:
To have a large enough texture for adding in the instancing bits that were missing before, mostly. The test pixel locations are taken with one pixel from each instanced triangle.